### PR TITLE
tests: full cli coverage

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

File: invenio_db/core.py [hunk 1]
```diff
@@ -44,11 +44,11 @@ def __init__(self, app=None, **kwargs):
 
     def init_app(self, app, **kwargs):
         """Initialize application object."""
-        self.init_db(app)
+        self.init_db(app, **kwargs)
         app.extensions['invenio-db'] = self
         app.cli.add_command(db_cmd)
 
-    def init_db(self, app):
+    def init_db(self, app, entry_points='invenio_db.models', **kwargs):
```
Comment (lnielsen) [inline invenio_db/core.py]: ``entry_points=['invenio_db.models']`` or ``entry_point='invenio_db.models'``? singular vs plural

File: .travis.yml [hunk 2]
```diff
@@ -45,6 +51,7 @@ python:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
+  - "travis_retry pip install psycopg2 pymysql"
```
Comment (lnielsen) [inline .travis.yml]: Perhaps add to setup.py? ``pip install invenio-db[postgresql]``

File: invenio_db/core.py (hunk 1 above)
Comment (jirikuncar) [inline invenio_db/core.py]: what about `entrypoint_name` like in `jsonresolver`?

**Comment by lnielsen** [comment]
:shipit: (i.e not necessary to fix the two tiny comments)

**Pull request merged**